src/plugins/workspace/public/components/workspace_form/constants.ts

```diff
@@ -18,6 +18,12 @@ export enum WorkspacePermissionItemType {
   Group = 'group',
 }
 
+export enum WorkspacePrivacyItemType {
+  PrivateToCollaborators = 'privat-to-collaborators',
```

Comment:
Typo: Shall we use `private-to-collaborators`?

Reply:
Sorry, there's a spelling mistake